sanic/request.py

```diff
@@ -166,9 +166,16 @@ def parse_multipart_form(body, boundary):
 
         post_data = form_part[line_index:-4]
         if file_name or file_type:
-            files[field_name] = File(
-                type=file_type, name=file_name, body=post_data)
+            file = File(type=file_type, name=file_name, body=post_data)
+            if field_name in files:
+                files[field_name].append(file)
+            else:
+                files[field_name] = [file]
         else:
-            fields[field_name] = post_data.decode('utf-8')
+            value = post_data.decode('utf-8')
+            if field_name in fields:
```

Comment:
Yes, but the purpose of this PR is to change the type of `files` and `fields` to `RequestParameters`.

The `files` and `fields` is not a instance of `defaultdict`.
I agree with you. It would be better to use the `defaultdict`. 
But in this case we need to convert `defaultdict` object to `RequestParameters`.

Are you sure that it would be better? (+ 1 type conversion)